lib/ZwaveClient.js

```diff
@@ -374,12 +374,13 @@ function onNodeDead (zwaveNode, oldStatus) {
 
 function onNodeValueAdded (zwaveNode, args) {
   logger.info(
-    `Node ${zwaveNode.id}: value added: ${getValueID(args)} =>`,
-    args.newValue
+    `Node ${zwaveNode.id}: adding value ${getValueID(args)} => ${args.newValue}`
   )
-
   // handle node values added 'on fly'
-  if (zwaveNode.ready) {
+  if (zwaveNode.ready || args.newValue) {
```

Comment:
This could be break when `newValue` is `0`.

Reply:
if i change it to `args.newvalue !== 'undefined'` would be better?